lib/PrimitiveHelpers.js

```diff
@@ -21,6 +21,13 @@ module.exports = {
     transformPrimitives : transformPrimitives
 };
 
+/**
+ * Compare two primitives to check if they share an arrtibute.
```

Comment:
Typo. Also, we want to check if they share an attribute accessor/data and that should probably be stated more explicitly.